scripts/startup/nuc_startup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Steps for NUC
+# 1) Start roscore
+# 2) Run nuc-eth-listener
+# 3) Run local pathfinding
+# 4) Run mocks scripts (MOCK_AIS ; MOCK_sensors)
+# No need to start visualizer automatically
+
+NT_ROOT="/home/raye/network-table"
+
+# nuc-eth-listener arguments
+NE_IP="192.168.1.60"
+NE_PORT="5555"
+
+NE_CMD="./build/bin/nuc_eth_listener $NE_IP $NE_PORT"
+
+# Assuming melodica is sourced properly
+
+printf "\nStartup script for NUC\n"
+printf "\n================================\n"
+
+# Roscore
+printf "Roscore\n"
+printf ".....................\n"
+
+if pgrep -f "roscore" > /dev/null;
+then
+    ROS_PID=$(pgrep -f "roscore")
+    printf "Roscore already running with PID: %d\n"$ROS_PID
+else
+    printf "Roscore not started. Starting roscore\n"
+
+    roscore
+    ROS_PID=$(pgrep -f "roscore")
+
+    printf "Started roscore with PID: %d\n" $ROS_PID
+fi
+printf "\n================================\n"
+
+# nuc-eth-listener
+printf "NUC ETH Listener\n"
+printf ".....................\n"
+
+if pgrep -f "nuc_eth_listener" > /dev/null;
+then
+    NE_PID=$(pgrep -f "nuc_eth_listener")
+    printf "NUC ETH listener already running with PID: %d\n" $NE_PID
+else
+    printf "NUC ETH listener not started. Starting new process\n"
+
+    cd $NT_ROOT
+    eval "$NE_CMD"
+    NE_PID=$(pgrep -f "nuc_eth_listener")
+
+    printf "NUC ETH listener started with PID: %d\n" $NE_PID
+fi
+printf "\n================================\n"
+
+# local pathfinding
+printf "Local Pathfinding\n"
+printf ".....................\n"
+
+if pgrep -f "local_pathfinding" > /dev/null;
+then
+  LP_PID=$(pgrep -f "local_pathfinding")
+  printf "Local pathfinding already running with PID: %d\n" $LP_PID
+else
+  printf "Local pathfinding not started. Starting new process\n"
+
+  rosrun local_pathfinding main_loop_and_interface.launch
```

Comment:
Can you pls change to:
`roslaunch local_pathfinding main_loop_and_interface.launch wandb:=true land_mass_file:=land_masses/vancouver_depth_5m_10_11_2021-10_28_14.csv
`

thank - this is just with the wandb logging